contracts/ForkDelta.sol

```diff
@@ -37,46 +43,71 @@ contract ForkDelta {
     depositingTokenFlag = false;
   }
 
+  /// This default function prevents people sending Ether directly into the contract.
   function() public {
     revert();
   }
 
+  /// Changes the official admin user address. Accepts Ethereum address.
   function changeAdmin(address admin_) public isAdmin {
     admin = admin_;
   }
 
+  /// Changes the account address that receives trading fees. Accepts Ethereum address.
   function changeFeeAccount(address feeAccount_) public isAdmin {
     feeAccount = feeAccount_;
   }
 
+  /// Changes the fee on makes. Can only be changed to a value less than it is currently set at.
   function changeFeeMake(uint feeMake_) public isAdmin {
     require (feeMake_ <= feeMake);
     feeMake = feeMake_;
   }
 
+  /// Changes the fee on takes. Can only be changed to a value less than it is currently set at.
   function changeFeeTake(uint feeTake_) public isAdmin {
     require(feeTake_ <= feeTake);
     feeTake = feeTake_;
   }
 
+  /// Changes the date that trades are free until. Accepts UNIX timestamp.
   function changeFreeUntilDate(uint freeUntilDate_) public isAdmin {
     freeUntilDate = freeUntilDate_;
   }
 
+  /**
+  * Note: With the payable modifier, this function accepts Ether.
+  * This function handles deposits of Ether into the contract.
+  * Emits a Deposit event.
+  */
   function deposit() public payable {
     tokens[0][msg.sender] = tokens[0][msg.sender].add(msg.value);
     emit Deposit(0, msg.sender, msg.value, tokens[0][msg.sender]);
   }
 
+  /**
+  * This function handles withdrawals of Ether from the contract.
+  * Verifies that the user has enough funds to cover the withdrawal.
+  * Emits a Withdraw event.
+  * @param amount: uint of the amount of Ether the user wishes to withdraw
+  */
   function withdraw(uint amount) public {
     require(tokens[0][msg.sender] >= amount);
     tokens[0][msg.sender] = tokens[0][msg.sender].sub(amount);
     msg.sender.transfer(amount);
     emit Withdraw(0, msg.sender, amount, tokens[0][msg.sender]);
   }
 
+  /**
+  * IMPORTANT: Remember to call Token(address).approve(this, amount) or this contract will not be able to do the transfer on your behalf.
```

Comment:
I would not make this note so prominent. The approve is on our UI code and that's about it.